api-conversion/src/main/java/io/strimzi/kafka/api/conversion/converter/APIConversionFailedException.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.api.conversion.converter;
+
+/**
+ * Exception used to indicate an conversion issue
+ */
+public class APIConversionFailedException extends RuntimeException {
```

Comment:
Personally I prefer the capitalization `ApiConversionFailed`, but I feel I'm fighting a losing battle with the community on this.

Reply:
Renamed.